libcontainer/configs/device.go

```diff
@@ -38,7 +38,13 @@ type Device struct {
 }
 
 func (d *Device) CgroupString() string {
-	return fmt.Sprintf("%c %s:%s %s", d.Type, deviceNumberString(d.Major), deviceNumberString(d.Minor), d.Permissions)
+	var p string
```

Comment:
The spec moved permissions out... Now these need to come from some new whitelist...